src/Compilers/VisualBasic/Portable/BoundTree/Expression.vb

```diff
@@ -1018,12 +1030,18 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End Get
         End Property
 
-        Private ReadOnly Property IConversionExpression_IsExplicit As Boolean Implements IConversionExpression.IsExplicit
+        Private ReadOnly Property IConversionExpression_ExplicitCastInSource As Boolean Implements IConversionExpression.ExplicitCastInSource
             Get
                 Return Me.ExplicitCastInCode
             End Get
         End Property
 
+        Private ReadOnly Property IConversionExpression_Conversion As IConversion Implements IConversionExpression.Conversion
+            Get
+                Return New Conversion(Me.ConversionKind, method:=Nothing)
```

Comment:
This is not correct and you will not be able to merge your change with incorrect implementation. Remember, you will have to test the change and make sure implementation is correct.